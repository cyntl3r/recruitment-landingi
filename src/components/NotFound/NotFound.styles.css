:root {
  font-size: 10px;
}

* {
  box-sizing: border-box;
  border: 0;
}

html,
body {
  margin: 0;
  padding: 0;
}

body {
  font-family: 'Open Sans', sans-serif;
  font-size: 1.2rem;
  color: #000000;
}

main {
  max-width: 900px;
  margin: 10vh auto 0 auto;
}
