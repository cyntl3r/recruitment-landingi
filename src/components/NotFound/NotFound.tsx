import { FunctionComponent } from 'react';
import { usePageTitle } from '../../hooks/usePageTitle';
import './NotFound.styles.css';

export const NotFound: FunctionComponent = () => {
  usePageTitle('Not Found');

  return (
    <p data-testid="not-found">
      A website with such an address has not been found!
    </p>
  );
};
