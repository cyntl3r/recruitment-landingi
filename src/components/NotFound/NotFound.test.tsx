import { render } from '../../testing/test-utils';
import { NotFound } from './NotFound';

test('snapshot', () => {
  const { container } = render(<NotFound />);
  expect(container).toMatchSnapshot();
});

test('wrapper render', () => {
  const { getByTestId } = render(<NotFound />);
  expect(getByTestId('not-found')).toBeDefined();
});

test('set page title', () => {
  render(<NotFound />);
  expect(document.title).toBe('Blog - Not Found');
});
