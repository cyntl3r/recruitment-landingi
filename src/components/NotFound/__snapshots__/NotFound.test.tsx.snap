// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <p
    data-testid="not-found"
  >
    A website with such an address has not been found!
  </p>
</div>
`;
